src/main/java/com/minecolonies/coremod/colony/ColonyView.java

```diff
@@ -541,9 +541,16 @@ public IMessage handleColonyViewMessage(@NotNull final ByteBuf buf, @NotNull fin
 
         if (buf.readBoolean())
         {
-            final NBTTagCompound compound = ByteBufUtils.readTag(buf);
-            this.requestManager = new StandardRequestManager(this);
-            this.requestManager.deserializeNBT(compound);
+            try
+            {
+                final NBTTagCompound compound = ByteBufUtils.readTag(buf);
+                this.requestManager = new StandardRequestManager(this);
+                this.requestManager.deserializeNBT(compound);
+            }
+            catch (RuntimeException e)
+            {
+                this.requestManager = new StandardRequestManager(this);
+            }
```

Comment:
Honestly, why create a new one if it crashes?
I'd just not assign a new one.

Reply:
And can we write trigger the rs reset message here if this fails? We don't want to avoid the crash we want to clean it up as well

Reply:
its not solving the NBT issue btw, just the clientsided crash. The server is still going to keep sending too large packages

Reply:
the problem is I did not found any method to calculate size of NBTTagCompound, I'm aware of it's not the best solution - so you want me to send a RS reset message?

Reply:
yeah, so we just send a reset on failure of this.